DedicatedServer/Scripts/Libs/Zrx/ModeLibs/Common/Commands/ModeCommands.Script.txt

```diff
@@ -1,7 +1,7 @@
 #Include "TextLib" as TL
 #Include "Libs/Zrx/ModeLibs/Common/Commands/CommandUtils.Script.txt" as CommandUtils
 
-#Const C_UIPosition_Y							65.
+#Const C_UIPosition_Y							20.
```

Comment:
```suggestion
#Const C_UIPosition_Y							50.
```

Will conflict with BigMessages, but that should not be too big of a problem. ModeCommands is displayed above BigMessages. Also, I think neither EvoSC, nor PyPlanet displays anything there.

![image](https://user-images.githubusercontent.com/53338174/148600234-69bed9a1-5bf9-4245-860d-a2df30b7dbb5.png)

Reply:
I approximately checked the position of the map info in both server controllers and it appears they do not conflict here. So this position seems okay 👍